pages/about.html

```diff
@@ -1,12 +1,33 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my cool portfolio</title>
+    <!-- Link to Style Sheet -->
+    <link href="/styles/main-style.css" rel="stylesheet" />
+    <link href="/styles/about-me-style.css" rel="stylesheet" />
 </head>
-<body>
+<body>  
+  <header>
+    <nav>
+
+      <a href="about.html">about me</a> 
+      <a href="portfolio.html">portfolio</a> 
+      <a href="index.html">home</a>
+      
+    </nav>
+  </header>
+  <main>
+    <section class="name_container">
+    <img src="/images/its-me.jpg" alt="cute picture of me holding plants" class="about_me_image">
   
+    <div id="about_me_box"><p>Means, loosely, 25% 75%. Except that those percentage values are much more firm than fractional units are. For example, if you added padding to those percentage-based columns, now you’ve broken 100% width (assuming a content-box box model). Fractional units also much more friendly in combination with other units, as you can imagine:</p></div>
+    </section>
+
+  </main>
+  <footer><div class="bottom">(c) philomena</div></footer>
```

Comment:
Careful with using `div` tags as a text container. They are used to divide collections of content.

Instead, let's use a `p` tag or other elements that are text-specific